### ISSUE TITLE
document convert error

### ISSUE THREAD

**Issue by cdzhoubin** [crawljax/crawljax#468]
log message:
2015-04-02 11:58:21,233 WARN  browser.WebDriverBackedEmbeddedBrowser - Could not get the dom
com.crawljax.core.CrawljaxException: Could not tranform the DOM
	at com.crawljax.util.DomUtils.getDocumentToString(DomUtils.java:239)
	at com.crawljax.browser.WebDriverBackedEmbeddedBrowser.getStrippedDom(WebDriverBackedEmbeddedBrowser.java:350)
	at com.crawljax.core.Crawler.crawlIndex(Crawler.java:449)
	at com.crawljax.core.CrawlTaskConsumer.crawlIndex(CrawlTaskConsumer.java:85)
	at com.crawljax.core.CrawlController.call(CrawlController.java:66)
	at com.crawljax.core.CrawljaxRunner.call(CrawljaxRunner.java:37)
	at com.sczbbx.common.scriptCrawl.CrawljaxHelper.runCrawljaxController(CrawljaxHelper.java:80)
	at com.sczbbx.common.scriptCrawl.CrawljaxHelper.runAjax(CrawljaxHelper.java:68)
	at com.sczbbx.common.scriptCrawl.CrawljaxHelper.runAjax(CrawljaxHelper.java:55)
	at com.sczbbx.common.scriptCrawl.CrawljaxHelperTest.testWithText12(CrawljaxHelperTest.java:26)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.apache.maven.surefire.junit4.JUnit4TestSet.execute(JUnit4TestSet.java:45)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:123)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:104)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:164)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:110)
	at org.apache.maven.surefire.booter.SurefireStarter.invokeProvider(SurefireStarter.java:175)
	at org.apache.maven.surefire.booter.SurefireStarter.runSuitesInProcessWhenForked(SurefireStarter.java:107)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:68)
Caused by: javax.xml.transform.TransformerException: java.lang.RuntimeException: 没有说明名称空间前缀 'font-size'。
	at com.sun.org.apache.xalan.internal.xsltc.trax.TransformerImpl.transform(TransformerImpl.java:752)
	at com.sun.org.apache.xalan.internal.xsltc.trax.TransformerImpl.transform(TransformerImpl.java:357)
	at com.crawljax.util.DomUtils.getDocumentToString(DomUtils.java:236)
	... 38 more
Caused by: java.lang.RuntimeException: 没有说明名称空间前缀 'font-size'。
	at com.sun.org.apache.xml.internal.serializer.SerializerBase.getNamespaceURI(SerializerBase.java:915)
	at com.sun.org.apache.xml.internal.serializer.SerializerBase.addAttribute(SerializerBase.java:431)
	at com.sun.org.apache.xalan.internal.xsltc.trax.DOM2TO.parse(DOM2TO.java:201)
	at com.sun.org.apache.xalan.internal.xsltc.trax.DOM2TO.parse(DOM2TO.java:230)
	at com.sun.org.apache.xalan.internal.xsltc.trax.DOM2TO.parse(DOM2TO.java:230)
	at com.sun.org.apache.xalan.internal.xsltc.trax.DOM2TO.parse(DOM2TO.java:230)
	at com.sun.org.apache.xalan.internal.xsltc.trax.DOM2TO.parse(DOM2TO.java:230)
	at com.sun.org.apache.xalan.internal.xsltc.trax.DOM2TO.parse(DOM2TO.java:230)
	at com.sun.org.apache.xalan.internal.xsltc.trax.DOM2TO.parse(DOM2TO.java:230)
	at com.sun.org.apache.xalan.internal.xsltc.trax.DOM2TO.parse(DOM2TO.java:230)
	at com.sun.org.apache.xalan.internal.xsltc.trax.DOM2TO.parse(DOM2TO.java:230)
	at com.sun.org.apache.xalan.internal.xsltc.trax.DOM2TO.parse(DOM2TO.java:230)
	at com.sun.org.apache.xalan.internal.xsltc.trax.DOM2TO.parse(DOM2TO.java:230)
	at com.sun.org.apache.xalan.internal.xsltc.trax.DOM2TO.parse(DOM2TO.java:230)
	at com.sun.org.apache.xalan.internal.xsltc.trax.DOM2TO.parse(DOM2TO.java:136)
	at com.sun.org.apache.xalan.internal.xsltc.trax.DOM2TO.parse(DOM2TO.java:98)
	at com.sun.org.apache.xalan.internal.xsltc.trax.TransformerImpl.transformIdentity(TransformerImpl.java:699)
	at com.sun.org.apache.xalan.internal.xsltc.trax.TransformerImpl.transform(TransformerImpl.java:743)
	... 40 more

test content:
 <table  border="0" cellspacing="0" cellpadding="0" align="center"  class="main" style="margin-top:36px;" height="400px">
        <tr><td colspan="3" class="now"> <table cellSpacing="0" cellPadding="0" border="0"><TR><TD align="center" width="15"></TD><TD align="left"><FONT color="#000000" class=currentpostionfont><b>您现在的位置：</b></FONT><A href='/sczw/'><FONT color="#000000" class=currentpostionfont>首页</FONT></A><FONT color="#FF7E00"> >> <a href='/sczw/jyfwpt'><FONT color="#FF7E00" class=currentpostionfont>交易服务平台</font></a> >> <a href='/sczw/jyfwpt/005001'><FONT color="#FF7E00" class=currentpostionfont>工程建设</font></a> >> <a href='/sczw/jyfwpt/005001/005001001'><FONT color="#FF7E00" class=currentpostionfont>招标公告</font></a></FONT></TD></TR></table></td></tr>
        <tr>
            <td class="leftbgmm" valign="top" style="width:250px;">
           <div class="head3">
            <div class="s_t1" style="background-color:#fff;">交易服务平台</div>
          </div>
            <div class="s-con" style=" height:auto;background-color:#fff;">
               <TABLE  cellSpacing="0" cellPadding="0" width="250" border="0">
<tr><td height=46 onclick="OpenDiv('TD005001');" style="cursor:hand;padding-left:56px; font-size:14px; background-repeat:no-repeat;"  background="/sczw/Template/Default/images/leftmenu_02.jpg"><font color='#ffffff'  size="+1" style="font-weight:bold">工程建设</font></a></td></tr>
<tr><td style="padding-left:75px;" id="TD005001"><table border=0 width="100%">
<tr><td width="20"><img src="/sczw/Template/Default/images/leftmenu_03.gif"></td><td  height=46 ><a href="/sczw/jyfwpt/005001/005001001" target="_self">
<font color='#ff7e00'>招标公告</font></a></td></tr>
<tr><td width="20"><img src="/sczw/Template/Default/images/leftmenu_03.gif"></td><td  height=46 ><a href="/sczw/jyfwpt/005001/005001002" target="_self">
<font color='#666666'>变更公告</font></a></td></tr>
<tr><td width="20"><img src="/sczw/Template/Default/images/leftmenu_03.gif"></td><td  height=46 ><a href="/sczw/jyfwpt/005001/005001003" target="_self">
<font color='#666666'>中标结果</font></a></td></tr>
</table>
